src/affiliations/models.py

```diff
@@ -2,23 +2,49 @@
 
 # Third-party dependencies:
 from django.db import models
+from django.core.exceptions import ValidationError
 
 
 class Affiliation(models.Model):
     """Define the shape of an affiliation."""
 
-    affiliation_id: models.IntegerField = models.IntegerField()
+    type: models.CharField = models.CharField()
+    """
+    10000 ID. All affiliations will have this ID, however, some affiliations
+    will share this ID. Affiliations that share this ID will have different
+    curation_panel_ids.
+    """
+    affiliation_id: models.IntegerField = models.IntegerField(
+        help_text="10000 number ID"
+    )
+    """
+    4000 or 5000 ID. This ID can be null as independent groups will not have
```

Comment:
Should be 40,000 or 50,000, not 4,000 or 5,000. See https://github.com/ClinGen/gci-vci-aws/blob/dev/gci-vci-serverless/local-files/affiliations.json for examples.

Reply:
Updated! Thank you!